gremlin-dotnet/test/Gremlin.Net.IntegrationTest/Docs/Reference/GremlinVariantsTests.cs

```diff
@@ -79,6 +79,14 @@ public void SerializationGraphson2Test()
 // end::serializationGraphSon[]
         }
 
+        [Fact(Skip = "No Server under localhost")]
```

Comment:
Not that important, but you could also remove the test `SerializationGraphBinaryTest` above now as it isn't used any more in docs.